ViewModel/BrowserViewModel.swift

```diff
@@ -211,24 +211,9 @@ final class BrowserViewModel: NSObject, ObservableObject,
     private func restoreBy(tabID: NSManagedObjectID) {
         if let tab = try? Database.viewContext.existingObject(with: tabID) as? Tab {
             webView.interactionState = tab.interactionState
-            if AppType.isCustom {
```

Comment:
This is where we were trying to fix the URL that was already set on the WebView from interactionState data.
Since we fix the URL in the data itself, this is no longer needed.